releases/latest/edge/bundle-airgap.yaml

```diff
@@ -0,0 +1,260 @@
+bundle: kubernetes
+name: kubeflow
+applications:
+  admission-webhook:
+    charm: ./admission-webhook_98aac65.charm
+    scale: 1
+    trust: true
+    resources:
+      oci-image: 172.17.0.2:5000/kubeflownotebookswg/poddefaults-webhook:v1.7.0
+  argo-controller:
+    charm: ./argo-controller_b59eaec.charm
+    scale: 1
+    resources:
+      oci-image: 172.17.0.2:5000/argoproj/workflow-controller:v3.3.8
+    options:
+      executor-image: 172.17.0.2:5000/argoproj/argoexec:v3.3.8
+  argo-server:
+    charm: ./argo-server_6d22972.charm
+    scale: 1
+    resources:
+      oci-image: 172.17.0.2:5000/argoproj/argocli:v3.3.8
+  dex-auth:
+    charm: ./dex-auth_f0211e2.charm
+    scale: 1
+    trust: true
+    resources:
+      oci-image: 172.17.0.2:5000/dexidp/dex:v2.36.0
+  istio-ingressgateway:
+    charm: ./istio-gateway_ubuntu-20.04-amd64.charm
+    scale: 1
+    trust: true
+    options:
+      kind: ingress
+      proxy-image: 172.17.0.2:5000/istio/proxyv2:1.17.3
```

Comment:
Let's expose here the `gateway_service_type` here with default value of node-port, since we are not sure whether airgapped cluster will have a LoadBalancer.

This was nicely caught by @ColmBhandal and @orfeas-k